packages/angular/src/builders/utils/esbuild-index-html.ts

```diff
@@ -26,7 +25,9 @@ export function indexHtml(
       replaceInFile(filePath, raw);
     });
     if (ssr) {
-      replaceInFile(resolve(outputDir, "server/index.server.html"), raw);
+      glob.sync(`${outputDir}/server/**/index.server.html`).forEach((filePath) => {
+        replaceInFile(filePath, raw);
+      });
```

Comment:
I'm not super familiar with the whole angular options system, so I went with a fix similar to how all the HTML files are handled in the `browser` folder.

However, I'm not sure if it wouldn't better to drop the `if (ssr)` entirely & generalize the first `glob` to `${outputDir}/**/*.html`

Reply:
feel free to resolve this if you're happy with the change to move forward with the PR!